tests/tests_app/cli/test_cp.py

```diff
@@ -47,7 +47,7 @@ def test_cp_local_to_remote(tmpdir, monkeypatch):
     )
 
     result = MagicMock()
-    result.get.return_value = V1UploadProjectArtifactResponse(upload_url="http://foo.bar")
+    result.get.return_value = V1UploadProjectArtifactResponse(upload_id="http://foo.bar")
```

Comment:
see my comment above for why this test doesn't test the reality

Reply:
done .